app/controllers/comments_controller.rb

```diff
@@ -10,7 +10,7 @@ def edit
     def create
       @comment = @article.comments.new(comment_params)
       if @comment.save
-        flash[:success] = "Careful now Ned"
+        flash[:success] = "Pomyślnie utworzono komentarz"
         redirect_to article_path(@article)
       else
         raise "error"
```

Comment:
raise "error" to niezbyt dobry sposób na przekazywanie użytkownikowi ze coś poszło nie tak :]